client/src/static/css/login.scss

```diff
@@ -1,13 +1,16 @@
-$adesso-grey: #857971;
-$adesso-blue: #0275d8;
-$adesso-white: #f4f4f4;
+$appbackground2: #dfeef5;
```

Comment:
why not to use adesso-blue?

Reply:
weil das nicht gut aussieht mit roter Schrift drauf wenn die Fehlermeldung kommt, ist viel zu dunkel. im Mockup ist es auch das helle blau was sonst auch als Hintergrund verwendet wird.